webtransport/csp-fail.https.window.js

```diff
@@ -14,6 +14,18 @@ promise_test(async t => {
   document.head.appendChild(meta);
 
   let wt = new WebTransport(webtransport_url('custom-response.py?:status=200'));
-  await promise_rejects_dom(t, 'SecurityError', wt.ready, 'ready promise should be rejected');
-  await promise_rejects_dom(t, 'SecurityError', wt.closed, 'closed promise should be rejected');
+
+  // Sadly we cannot use promise_rejects_dom as the error constructor is
```

Comment:
Is there any chance `promise_reject_js` could work?
https://web-platform-tests.org/writing-tests/testharness-api.html?highlight=promise_reject#promise_rejects_js